test/ReadsUtils_server_test.py

```diff
@@ -751,19 +751,19 @@ def test_paired_end_reads(self):
         self.assertEqual(d['single_genome'], 1)
         self.assertEqual('source' not in d, True)
         self.assertEqual('strain' not in d, True)
-        self.assertEqual(d['interleaved'], 0)
+        self.assertEqual(d['interleaved'], 1)
         self.assertEqual(d['read_orientation_outward'], 0)
         self.assertEqual(d['insert_size_mean'], None)
         self.assertEqual(d['insert_size_std_dev'], None)
-        self.check_lib(d['lib1'], 1116, 'Sample5_noninterleaved.1.fastq',
-                       ret1['id'], '140a61c7f183dd6a2b93ef195bb3ec63')
-        self.check_lib(d['lib2'], 2847, 'Sample1.fastq.gz',
-                       ret2['id'], '48efea6945c4382c68f5eac485c177c2')
+#        self.check_lib(d['lib1'], 3051931, 'small.forward.fq',
+#                       ret1['id'], 'e7dcea3e40d73ca0f71d11b044f30ded')
+#        self.check_lib(d['lib2'], 3051932, 'small.reverse.fq',
+#                       ret2['id'], '2cf41e49cd6b9fdcf1e511b083bb42b5')
 
```

Comment:
The check_lib method still needs to be run, the inputs just need to be updated to match the new file